### PR TITLE
Patch

### PR DESCRIPTION
**Pull request by thowell**

### CODE REVIEW

File: right_hand.xml.patch
Comment (alberthli) [inline right_hand.xml.patch:1]: Stray empty file?

**Review by alberthli — CHANGES_REQUESTED** [review]
besides stray file, lgtm

File: right_hand.xml.patch
Comment (thowell) [reply right_hand.xml.patch:1]: Good catch!

**Review by alberthli — APPROVED** [review]
nice catch on not loading the modified model into `task.xml`. lgtm.

**Pull request merged**